Controllers/UserController.cs

```diff
@@ -0,0 +1,62 @@
+using Microsoft.AspNetCore.Mvc;
+using QSWebAPI.Models;
+using QSWebAPI.Services.Base;
+
+namespace QSWebAPI.Controllers
+{
+    /// <summary>
+    ///     Controller which takes care of the User-domain
+    /// </summary>
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
```

Comment:
We don't know any of the specifications for a User yet, nor the way we want to store it. I would refrain from implementing domain logic we haven't discussed with the group yet. This has  to be done within the selection of the user stories (hopefully sprint #2).